tests/func/test_datachain.py

```diff
@@ -1366,3 +1372,68 @@ def file_info(file: File) -> DataModel:
         ],
         "file_info__path",
     )
+
+
+def test_to_from_csv_remote(cloud_test_catalog):
+    ctc = cloud_test_catalog
+    path = f"{ctc.src_uri}/test.csv"
+
+    df = pd.DataFrame(DF_DATA)
+    dc_to = DataChain.from_pandas(df, session=ctc.session)
+    dc_to.to_csv(path)
+
+    dc_from = DataChain.from_csv(path, session=ctc.session)
+    df1 = dc_from.select("first_name", "age", "city").to_pandas()
+    assert df1.equals(df)
+
+    # Cleanup any written files
```

Comment:
qq - are we using real clouds? do we care about cleanup here? if we care - should it be wrapped into a fixture (so that we do this even in case of a failure)

Reply:
This uses simulated clouds provided by `pytest-servers` - and the cleanup code is necessary to prevent test failures (as these uploaded files are not automatically cleaned up). I moved the cleanup code into a fixture to keep it in one location.